addons/core/XEH_preInit.sqf

```diff
@@ -284,8 +268,190 @@ If (isServer && {isMultiplayer || is3DENMultiplayer}) then {
         TFAR_setting_defaultFrequencies_lr_independent = str(TFAR_defaultFrequencies_lr_independent apply {parseNumber _x});
     }
 ] call CBA_Settings_fnc_init;
-
-
+[
+    "TFAR_giveMicroDagrToSoldier", 
+    "CHECKBOX", 
+    "STR_radio_give_microdagr_to_soldier", 
+    "Task Force Arrowhead Radio", 
+    true,
+    1
+] call CBA_Settings_fnc_init;
+[
+    "TFAR_givePersonalRadioToRegularSoldier", 
+    "CHECKBOX", 
+    "STR_radio_give_personal_radio_to_regular_soldier", 
+    "Task Force Arrowhead Radio", 
+    false,
+    1
+] call CBA_Settings_fnc_init;
+[
+    "TFAR_setting_DefaultRadio_Rifleman_West", 
+    "EDITBOX", 
+    ["Default Rifleman radio","Default RiflemanRadio west description"], 
+    "Task Force Arrowhead Radio", 
+    "TFAR_rf7800str",
+    1,
+    {
+        TFAR_DefaultRadio_Rifleman_West = If (_this call TFAR_fnc_isPrototypeRadio) then {_this} else {"TFAR_rf7800str"};
+        publicVariable "TFAR_DefaultRadio_Rifleman_West";
```

Comment:
publicVariable here is useless. As that function will be executed on each client and server anyway.

Reply:
You're right, I've changed it.